slices/apache2.yaml

```diff
@@ -0,0 +1,319 @@
+package: apache2
+
+essential:
+  - apache2_copyright
+
+slices:
+  bins:
+    essential:
+      - apache2_configs
+      - apache2-bin_bins
+      - apache2-data_data
+      - apache2-utils_bins
+      - base-files_bin
+      - coreutils_bins
+      - media-types_data
+      - perl-base_bins
+      - perl-base_modules
+      - procps_bins
+    contents:
+      /usr/sbin/a2disconf:
+      /usr/sbin/a2dismod:
+      /usr/sbin/a2dissite:
+      /usr/sbin/a2enconf:
+      /usr/sbin/a2enmod:
+      /usr/sbin/a2ensite:
+      /usr/sbin/a2query:
+      /usr/sbin/apache2ctl:
+      /usr/sbin/apachectl:
+
+  configs:
+    contents:
+      /etc/apache2/apache2.conf:
+      /etc/apache2/conf-available/**:
```

Comment:
We need to create symlinks for the following confs (in /etc/apache2/conf-enabled/) that the postinst script are enabling by default:
```
                for conf in charset localized-error-pages other-vhosts-access-log \
                                security serve-cgi-bin ; do
                        a2enconf -m -q $conf
                done
```